server/src/HatoholArmPluginGateHAPI2.cc

```diff
@@ -593,9 +600,17 @@ struct HatoholArmPluginGateHAPI2::Impl
 
 		bool isSucceeded(JSONParser &parser)
 		{
+			ArmStatus &status = m_impl.m_armStatus;
+			ArmInfo armInfo = status.getArmInfo();
```

Comment:
I understood what you want, but I'm not agree this patch.
Because `isSucceeded` should have one functionality which is checking response JSON `result` key's value.
If you want to set armInfo status in this method, we should use another name for this, like `validateSucceeded` or similar method name.